srcpkgs/vagrant/template

```diff
@@ -10,8 +10,9 @@ short_desc="Tool for building and distributing development environments"
 maintainer="Andrea Brancaleoni <abc@pompel.me>"
 license="MIT"
 homepage="https://www.vagrantup.com/"
-distfiles="https://github.com/mitchellh/${pkgname}/archive/v${version}.tar.gz"
-checksum=4f0e6b1d466e26dead682c4d4843e8f64a012eba4be91506ae6c6d34d3d9c8f9
+changelog="https://github.com/hashicorp/${pkgname}/blob/v${version}/CHANGELOG.md"
+distfiles="https://github.com/hashicorp/${pkgname}/archive/v${version}.tar.gz"
```

Comment:
expand pkgname

Reply:
Do you mean to use `vagrant` instead of the variable `${pkgname}`?

Reply:
yes